site2/website/blog/2021-12-21-apache-pulsar-hits-10000-github-stars.md

```diff
@@ -0,0 +1,58 @@
+
+Recent Pulsar project updates have brought new capabilities to the project. Below we look at key releases and launches.  
+
+## 1. Pulsar 2.8 - Unified Messaging and Streaming with Transactions
+
+Pulsar 2.8 (released in June 2021) introduced many major updates. The Pulsar Transaction API was added to support atomicity across multiple topics and enable end-to-end exactly-once message delivery guarantee for streaming jobs. Replicated subscriptions was added to enhance Pulsar’s geo-replication. With this feature, a consumer can restart consuming from the failure point in a different cluster in case of failover. 
+
+## 2. Function Mesh - Simplifying Complex Streaming Jobs in the Cloud
+
+Function Mesh is an ideal tool for those who are seeking cloud-native serverless streaming solutions. It is a Kubernetes operator that enables users to run Pulsar Functions and connectors natively on Kubernetes, unlocking the full power of Kubernetes’ application deployment, scaling, and management. Function Mesh is also a serverless framework used to orchestrate multiple Pulsar Functions and I/O connectors for complex streaming jobs in a simple way.
+
+## 3. Pulsar Connectors - AWS SQS Connector, Cloud Storage Sink Connector, and More
+
+Pulsar connectors enable easy integration between Pulsar and external systems with added benefits. For instance, the AWS SQS Connector enables secure integration between Pulsar and SQS without needing to write any code. And the Cloud Storage Sink Connector can export data by guaranteeing exactly-once delivery semantics to its consumers. It provides applications that export data from Pulsar, the benefits of fault tolerance, parallelism, elasticity, and much more. 
+
+## 4. Protocol Handlers - Kaffa-on-Pulsar, MQTT-on-Pulsar, and More
+
+Protocol handlers, such as Kaffa-on-Pulsar (KoP) and MQTT-on-Pulsar, allow Pulsar to interact with applications built on other messaging platforms, lowering the barrier to Pulsar adoption. KoP became production-ready since KoP 2.8 and MoP enables MQTT applications to leverage Pulsar’s infinite event stream retention with BookKeeper and tiered storage.
+
+# What’s Next
+
+In the upcoming Pulsar 2.9 release, you can expect the following updates: 
+
+* Introducing a pluggable metadata interface for ZooKeeper metadata management to improve consistency, resilience, and stability, and reduce technical debt. 
+* Launching the Oracle Debezium Connector and the schema-aware Elasticsearch Sink Connector.
+* Adding the ability to run Kafka Connect sinks as Pulsar sinks.
+
+The Pulsar community is planning more features for future Pulsar releases, including removing ZooKeeper, autoscaling topics, and simplified management of document-based policy. 
+
+# Get Involved
+
+* Pulsar 2.8.1 was released in September 2021. [Download it now](https://pulsar.apache.org/en/download/) and try it out! 
+* Join the [Pulsar community on Slack](https://apache-pulsar.herokuapp.com/).
```

Comment:
I think we should mention subscribing to the users or dev mailing lists (in addition to the joining the slack group), as those are the official ways to get involved in the community.

Reply:
Note that a reference to https://pulsar.apache.org/en/contact/ would include all of the ways to get connected to the community.

Reply:
I have add a CTA to the mailing lists. Thanks!